mappings/net/minecraft/network/listener/ServerPlayPacketListener.mapping

```diff
@@ -1,12 +1,22 @@
 CLASS nd net/minecraft/network/listener/ServerPlayPacketListener
+	METHOD a onTeleportConfirm (Lne;)V
 	METHOD a onChatMessage (Lng;)V
+	METHOD a onClientStatus (Lnh;)V
 	METHOD a onClientSettings (Lni;)V
 	METHOD a (Lnl;)V
```

Comment:
Similarly, are there any other methods here which could be remapped as we've packet names now?

Reply:
I'll leave the one attached to `UnlockRecipesClientPacket` as-is for now until I map `lt`. Since `lt` appears to match https://wiki.vg/Protocol#Unlock_Recipes closer than the one actually mapped as `UnlockRecipesClientPacket` I'll probably change the name of that packet

Reply:
`UnlockRecipesClientPacket` -> `SelectAdvancementTabClientPacket`
`onUnlockRecipes` -> `onSelectAdvancementTab`

`lt` -> `UnlockRecipesClientPacket`